platforms/winpack_dldt/build_package.py

```diff
@@ -0,0 +1,447 @@
+
+        log.info('DLDT build completed')
+
+
+    def make_sysroot(self):
+        sysroot_bin_dir = prepare_dir(self.sysrootdir / 'bin', create=True)
+        copytree_smart(self.build_dir / 'install', self.sysrootdir / 'ngraph')
+        #rm_one(self.sysrootdir / 'ngraph' / 'lib' / 'ngraph.dll')
+
+        build_bin_dir = self.build_dir / 'bin' / 'intel64' / 'Release'
+        def copy_bin(name):
+            copytree_smart(build_bin_dir / name, sysroot_bin_dir / name)
+        def copy_dll(name):
+            copy_bin(name + '.dll')
+            copy_bin(name + '.pdb')
+        copy_bin('cldnn_global_custom_kernels')
+        copy_bin('cache.json')
+        copy_dll('clDNNPlugin')
+        copy_dll('HeteroPlugin')
+        copy_dll('inference_engine')
+        copy_dll('inference_engine_nn_builder')
+        copy_dll('MKLDNNPlugin')
+        copy_dll('myriadPlugin')
+        copy_dll('ngraph')
+        copy_bin('plugins.xml')
+        copytree_smart(self.build_dir / 'bin' / 'intel64' / 'pcie-ma248x.elf', sysroot_bin_dir / 'pcie-ma248x.elf')
+        copytree_smart(self.build_dir / 'bin' / 'intel64' / 'usb-ma2x8x.mvcmd', sysroot_bin_dir / 'usb-ma2x8x.mvcmd')
+        copytree_smart(self.build_dir / 'bin' / 'intel64' / 'usb-ma2450.mvcmd', sysroot_bin_dir / 'usb-ma2450.mvcmd')
+
+        copytree_smart(self.srcdir / 'inference-engine' / 'temp' / 'tbb' / 'bin', sysroot_bin_dir)
+        copytree_smart(self.srcdir / 'inference-engine' / 'temp' / 'tbb', self.sysrootdir / 'tbb')
+
+        sysroot_ie_dir = prepare_dir(self.sysrootdir / 'deployment_tools' / 'inference_engine', create=True)
+        sysroot_ie_lib_dir = prepare_dir(sysroot_ie_dir / 'lib' / 'intel64', create=True)
+
+        copytree_smart(self.srcdir / 'inference-engine' / 'include', sysroot_ie_dir / 'include')
+        copytree_smart(self.build_dir / 'install' / 'lib' / 'ngraph.lib', sysroot_ie_lib_dir / 'ngraph.lib')
+        copytree_smart(build_bin_dir / 'inference_engine.lib', sysroot_ie_lib_dir / 'inference_engine.lib')
+        copytree_smart(build_bin_dir / 'inference_engine_nn_builder.lib', sysroot_ie_lib_dir / 'inference_engine_nn_builder.lib')
+
+        sysroot_license_dir = prepare_dir(self.sysrootdir / 'etc' / 'licenses', create=True)
+        shutil_copy2(self.srcdir / 'LICENSE', sysroot_license_dir / 'dldt-LICENSE')
+        shutil_copy2(self.srcdir / 'ngraph/LICENSE', sysroot_license_dir / 'ngraph-LICENSE')
+        shutil_copy2(self.sysrootdir / 'tbb/LICENSE', sysroot_license_dir / 'tbb-LICENSE')
+
+        log.info('DLDT sysroot preparation completed')
+
+
+#===================================================================================================
+
+class Builder:
+    def __init__(self, config):
+        self.config = config
+        self.build_dir = prepare_dir(Path(self.config.output_dir) / 'opencv_build', create=True)
+        self.package_dir = prepare_dir(Path(self.config.output_dir) / 'package/opencv', create=True)
```

Comment:
Shouldn't `package_dir` be the same as `output_dir`? For now `output_dir` contains only `package` and `opencv_build`, if `opencv_build` will be moved to `.build_cache`, then whole hierarchy can be flattened (1-2 levels).